dfetch/commands/list.py

```diff
@@ -0,0 +1,44 @@
+"""List metadata of all projects listed in the manifest.
+
+For each project listed in the manifest the metadata is shown,
+such as timestamp of last fetch, revision, remote url, etc.
+"""
+
+import argparse
+import logging
+import os
+
+from colorama import Fore
+
+import dfetch.commands.command
+import dfetch.manifest.manifest
+from dfetch.project.metadata import Metadata
+
+
+logger = logging.getLogger(__name__)
```

Comment:
In a recent branch I've created a `DLogger` class. This moves all the colorama uglieness into a single place.

```python
from dfetch.log import get_logger

logger = get_logger(__name__)

logger.print_info(.., ..)
```